Test/Unittest/blackbox/flask/test_traveler_resource.py

```diff
@@ -13,13 +13,16 @@
 _APP_CONFIG = {
     "repositories_config": {
         "repository_type": "memory",
-    }
+    },
+    "request_handlers_config": {
+        "request_handler_type": "rest"
+    },
```

Comment:
This may not be needed with the more recent change of specifying the handler type from the flask_app module. Should be checked.

Reply:
fixed